my-connections/src/java/org/sakaiproject/widgets/myconnections/ui/WidgetPage.java

```diff
@@ -69,6 +71,36 @@ public void onInitialize() {
 		
 		//log.error("********CURRENT USER ID: " + currentUserUuid);
 
+		// TODO: DRY this out
+		// get requests
+		List<Person> requests = this.connectionsLogic.getConnectionRequestsForUser(currentUserUuid);
+		
+		// sort
+		Collections.sort(requests, new Comparator<Person>() {
```

Comment:
Just a sort will suffice. The `BasicPerson` already has a comparator on displayname and `Person` extends it.